crates/next-custom-transforms/tests/full/vercel-flags/1/input.js

```diff
@@ -0,0 +1,4 @@
+
+import {flag }from '@vercel/flags/next'
+
+export const myFlag = flag()
```

Comment:
Could you add tests to ensure

- if there is already a `key` property on the object we don't override it
- if there is already an argument to `flags({})` we merge the key onto the object instead of replacing the whole argument